src/libcore/num/i128.rs

```diff
@@ -14,4 +14,4 @@
 
 #![unstable(feature = "i128", issue="35118")]
 
-int_module! { i128 }
+int_module! { i128, #[unstable(feature = "i128", issue="35118")] }
```

Comment:
They are two different features. One (`i128_type`) gates the actual type in the language, the other (`i128`) gates the library members. Here we use the library gating one.